.github/workflows/linter.yml

```diff
@@ -0,0 +1,13 @@
+name: Linter
```

Comment:
i'll try to make it easier and point out the new files that are actually worth looking over.

this is the file that sets up the linter on GitHub